doc/source/tutorial/examples/plot_interp_grid.py

```diff
@@ -0,0 +1,63 @@
+import math
+
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib import ticker
+
+
+fig, axes = plt.subplots(1, 2, figsize=(11, 5))
+orders = [2, 3]
```

Comment:
from the zip below I believe axes should have same size as order so consider
```
orders = [2, 3]
fig, axes = plt.subplots(1, len(orders), figsize=(11,5))
```

Reply:
thanks, I have made the suggested change